instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/bedrock.py

```diff
@@ -1004,3 +1018,9 @@ def on_error(
                 duration,
                 attributes=metrics_attributes,
             )
+
+    def _stream_has_output(self, response_body: dict[str, Any]):
+        return "output" in response_body
+
+    def _stream_has_content(self, response_body: dict[str, Any]):
+        return "message" in response_body["output"]
```

Comment:
```suggestion
    def _stream_has_output_content(self, response_body: dict[str, Any]):
        return "output" in response_body and "message" in response_body["output"]
```

What do you think on merging the two functions since the caller don't care much on having any intermediate ?

Reply:
Makes sense to me! Should've done it in the first place, but took your previous suggestion too literal 😄